src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/ExpanderTemplateSettings.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class ExpanderTemplateSettings : DependencyObject
+	{
+		public double ContentHeight
+		{
+			get => (double)GetValue(ContentHeightProperty);
+			set => SetValue(ContentHeightProperty, value);
+		}
+
+		public static DependencyProperty ContentHeightProperty { get; } =
```

Comment:
I know it's more verbose, but I think this should be broken up following conventions in WinUI. There should be separate `ExpanderTemplateSettings.Properties.cs` and `ExpanderTemplateSettings.cs` files as in PR #5626.